### PR TITLE
AMS David Bros

### PR DESCRIPTION
**Pull request by TheHunter896**
Done

### CODE REVIEW

**Comment by IronhackAMS** [comment]
Almost working mate, need a couple of small adjustments in the pepperoni and and green peppers department to make it work correctly:)

File: starter-code/pizza.js [hunk 1]
```diff
@@ -1 +1,119 @@
...
+
+function activatePeperoni(){
+    
+    $(".pep").toggle();
+    $("#pepperoni").toggle();
+    
+    if(values.statpep == true){
+        var aux = $("#strong").text();
+        var aux2 = parseInt(aux.slice(1,3), 10);
+        aux2 -= values.peperoni;
```
Comment (IronhackAMS) [inline starter-code/pizza.js:26]: try change this to +=

File: starter-code/pizza.js [hunk 2]
```diff
@@ -1 +1,119 @@
...
+        $("#strong").html("$"+ aux2.toString());
+        values.statpep = false;
+    }
+    else{
+        var aux = $("#strong").text();
+        var aux2 = parseInt(aux.slice(1,3), 10);
+        aux2 += values.peperoni;
```
Comment (IronhackAMS) [inline starter-code/pizza.js:33]: and here -=

**Comment by TheHunter896** [comment]
What? I tested it and it worked -.-

Sent from my iPhone

On 31 Oct 2018, at 21:45, IronhackAMS <notifications@github.com<mailto:notifications@github.com>> wrote:


@IronhackAMS commented on this pull request.

________________________________

In starter-code/pizza.js<https://github.com/ironhack-labs/lab-jquery-pizza-builder/pull/575#discussion_r229859893>:

> +    statmush: true,
+    statgpep: true,
+    statwhites: true,
+    statcrust: true,
+}
+
+
+function activatePeperoni(){
+
+    $(".pep").toggle();
+    $("#pepperoni").toggle();
+
+    if(values.statpep == true){
+        var aux = $("#strong").text();
+        var aux2 = parseInt(aux.slice(1,3), 10);
+        aux2 -= values.peperoni;


try change this to +=

________________________________

In starter-code/pizza.js<https://github.com/ironhack-labs/lab-jquery-pizza-builder/pull/575#discussion_r229859967>:

> +function activatePeperoni(){
+
+    $(".pep").toggle();
+    $("#pepperoni").toggle();
+
+    if(values.statpep == true){
+        var aux = $("#strong").text();
+        var aux2 = parseInt(aux.slice(1,3), 10);
+        aux2 -= values.peperoni;
+        $("#strong").html("$"+ aux2.toString());
+        values.statpep = false;
+    }
+    else{
+        var aux = $("#strong").text();
+        var aux2 = parseInt(aux.slice(1,3), 10);
+        aux2 += values.peperoni;


and here -=

**Pull request closed without merging**